index.html

```diff
@@ -19,12 +19,11 @@ <h2>Add an item</h2>
     </form>
 
     <h2>List of items</h2>
-    <table id="listBook" style="width:100%">
+    <table id="listBook" style="width:60%">
```

Comment:
I think it worth changing the table's id to 'listOfItems' or something. 'listBook' looks misleading. It's not for this change (branch). You can do it later as a separate change.